images/casl-ansible/Dockerfile

```diff
@@ -4,10 +4,13 @@ USER root
 
 # Update System and install clients
 RUN \
-    yum install -y --setopt=tsflags=nodocs https://repos.fedorapeople.org/repos/openstack/openstack-queens/rdo-release-queens-1.noarch.rpm; \
+    yum install -y --setopt=tsflags=nodocs \
+    https://repos.fedorapeople.org/repos/openstack/openstack-queens/rdo-release-queens-1.noarch.rpm \
```

Comment:
nitpick - for readability, can you please indent the file names as these aren't commands ... in that way, we have the commands (such as `yum`, `pip`, etc. at the same level).

Reply:
@jeffcpullen If you can get this one done, we can merge. Thank you.